Tests/SpeziSchedulerTests/SchedulerTests.swift

```diff
@@ -281,4 +281,22 @@ final class SchedulerTests: XCTestCase {
         let task = try JSONDecoder().decode(Task<String>.self, from: Data(json.utf8))
         XCTAssertEqual(task.schedule.calendar, .current)
     }
+    @MainActor func testEventHashEqualityForScheduledVsCompleted() throws {
```

Comment:
This should fix your swiftlint warning.

```suggestion
    @MainActor
    func testEventHashEqualityForScheduledVsCompleted() throws {
```